src/disease/schemas.py

```diff
@@ -88,6 +88,36 @@ class NamespacePrefix(Enum):
     WIKIDATA = "wikidata"
 
 
+# Source to URI (from identifiers.org, if found)
+NAMESPACE_TO_SYSTEM_URI: dict[NamespacePrefix, str] = {
```

Comment:
maybe this is my own misunderstanding, but shouldn't we be using pURLs where possible here?

Reply:
I think this is part of the ambiguity with this. I can update